src/symbol.ts

```diff
@@ -101,13 +101,15 @@ export function d3Symbol(symbol: SymbolType = d3SymbolCircle, fillStyle: string
  * @param size
  * @returns {function(CanvasRenderingContext2D): undefined}
  */
+
+const defaultOptions: ISymbolOptions = {
+  fillColor: 'steelblue',
+  selectedColor: 'red',
+  hoverColor: 'orange',
+  symbolSize: 20
+};
+
 export function circleSymbol(params?: ISymbolOptions): ISymbol<any> {
-  const defaultOptions: ISymbolOptions = {
-    fillColor: 'steelblue',
-    selectedColor: 'red',
-    hoverColor: 'orange',
-    symbolSize: 20
-  };
   const options = merge(defaultOptions, params || {});
```

Comment:
merge has the arguments: (target, source1, source2; ...) -> thus will change your defaultOptions